net/autossh/LICENSE

```diff
@@ -0,0 +1,24 @@
+Copyright (c) 2018 Verb Networks Pty Ltd <contact@verbnetworks.com>
```

Comment:
This file is implicit from the code lines. and will be in the main LICENSE file merged with all other contributors, but I don't have issues pulling this in. :)